ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 0.20.0
```

Comment:
Though this kind of a bug fix, it changes the API so I bumped the minor version

Reply:
Does it really change the API?

Reply:
Ahh yes, it does for Go. :(

Reply:
Actually all languages, unfortunately.   javascript` Promise<any>` -> `Promise<void>`, java `Object` -> `void`, python `MessageAttemptOut` -> `None` (which i think was actually always wrong)

Reply:
Yeah, but it's barely an API break. Will only break in weird scenarios, not under normal usage.